tools/travis/backend

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+# This script is very similar to tools/test-all (what one runs
+# locally).  Possibly they should be merged, though it's worth noting,
+# they are intentionally different (basically some slow stuff is not
+# worth running in `test-all`).
```

Comment:
Is this a suggestion to possibly make them behave the same (despite that slow stuff), or just to deduplicate the code?

If the latter, perhaps

>  This script is very similar to tools/test-all (what one runs locally). The intended difference is that we run here some slow stuff that's not worth running in `test-all`. Possibly the code should be deduplicated.